Cargo.toml

```diff
@@ -0,0 +1,30 @@
+[package]
+authors = ["omnect@conplement.de"]
+description = "WebService providing access to omnect device features."
+edition = "2021"
+homepage = "https://www.omnect.io/home"
+license = "MIT OR Apache-2.0"
+name = "omnect-ui"
+readme = "README.md"
+repository = "git@github.com:omnect/omnect-ui.git"
+version = "0.1.0"
+
+[dependencies]
+actix-files = "0.6"
+actix-rt = "2.9"
+actix-web = { version = "4", features = ["rustls-0_22"] }
+actix-web-httpauth = "0.8"
+anyhow = "1.0"
+env_logger = "0.8"
+http-body-util = { version = "0.1", default-features = false, features = [] }
+hyper = { version = "1.3", default-features = false, features = [
```

Comment:
hyper + actix?
in other crates we use 
request + actix.

Reply:
yes. only hyper provides the low level features to use unix sockets.